storage-proofs-porep/tests/halo2_stacked_circuit.rs

```diff
@@ -150,25 +175,51 @@ fn test_sdr_porep_circuit_2kib_halo2() {
     // Halo2 keygen, proving, and verifying are slow and consume a lot of memory, thus we only test
     // those for a small sector size circuit (the halo2 compound proof tests will run the halo2
     // prover and verifier for larger sector sizes).
+    init_logger();
```

Comment:
Maybe move logger initialization to the top of the inner test function so logging can be disabled via a single comment?